charts/cluster/templates/tests/ping.yaml

```diff
@@ -30,8 +30,13 @@ spec:
                 secretKeyRef:
                   name: {{ include "cluster.fullname" . }}-app
                   key: password
+            - name: PGDBNAME
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "cluster.fullname" . }}-app
```

Comment:
Not sure we can assume the database is always called this way 🤔

Reply:
The secret name assumption is already made in the ping.yaml. If you have suggestions to make it more dynamic, I am happy to try to fix it. 

However, the new assumption is for users to add the `dbname` secret key. I am going to fix it so that it will be optional. Note that if the user left the `initdb.secret` value empty, the `dbname` is already set.

Reply:
This comes from the operator @phisco. It's always `app` AFAIK, even if you change `dbname`.